eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -47,6 +48,21 @@ jobs:
       vmImage: $(OSVmImage)
 
     steps:
+      - ${{ if eq(parameters['AllocateResourceGroup'], 'true') }}:
```

Comment:
As we chatted I'm fine with this for now but I'd like to see this be a yml template step under eng/common that we can share across languages.

Reply:
https://github.com/Azure/azure-sdk-tools/issues/358#issuecomment-591544485